internal/services/bot/bot_web_app_resource.go

```diff
@@ -191,7 +192,17 @@ func resourceBotWebAppCreate(d *pluginsdk.ResourceData, meta interface{}) error
 		return fmt.Errorf("creating Web App Bot %q (Resource Group %q): %+v", resourceId.Name, resourceId.ResourceGroup, err)
 	}
 
-	// TODO: in 4.0 we should remove the "Default Site" on the Directline resource at this point if we can
```

Comment:
I'm not sure I agree with this change. I'm looking through the issues and not seeing anyone complaining about it. I'd be more inclined to remove the comment and we can address it if people have issues with it

Reply:
I like the idea of keeping it simpler. I just relocated the in line comment to a note in the docs.